napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -340,7 +369,32 @@ def _is_installed_with_conda():
         return False
 
 
+def is_conda_package(pkg):
+    """Determines if plugin was installed through conda.
+
+    Returns
+    -------
+    bool: True if a conda package, False if not
+    """
+    conda_meta_dir = Path(sys.prefix) / 'conda-meta'
+    try:
+        if conda_meta_dir.is_dir():
+            for fname in conda_meta_dir.iterdir():
+                if fname.suffix == '.json':
```

Comment:
Would love a comment here about what flag we're looking for (i.e. what is this json file) that indicates this package is a conda package.

Reply:
@ppwadhwa something we could add is:
```python
"""
Installed conda packages within a conda installation and environment can be identified as files
with the template `<package-name>-<version>-<build-string>.json` saved within a `conda-meta` folder within
the given environment of interest.
"""
```

Reply:
@ppwadhwa something we could add is:
```python
"""
Installed conda packages within a conda installation and environment can be identified as files
with the template `<package-name>-<version>-<build-string>.json` saved within a `conda-meta` folder within
the given environment of interest.
"""
```